phc-rails/db/schema.rb

```diff
@@ -37,20 +36,23 @@
     t.string   "How_long_have_you_been_homeless__c",        limit: 255
     t.string   "Where_do_you_usually_go_for_healthcare__c", limit: 255
     t.string   "Medical_Care_Other__c",                     limit: 255
+    t.date     "Birthdate__c"
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.